src/main/java/com/documaster/rms/noark/ws/client/fulltextsearch/FullTextRequest.java

```diff
@@ -0,0 +1,81 @@
+package com.documaster.rms.noark.ws.client.fulltextsearch;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import com.documaster.rms.noark.ws.client.Bean;
+
+public class FullTextRequest implements Bean {
+
+	private String doctype;
```

Comment:
The doctype could possibly be an enum of the possible values we support.

Reply:
Addressed with https://github.com/documaster/ws-client-n5-java/pull/33/commits/c81676523ff5005b53b30f622728181e67cfb2cf.